functions/function/image-recognition-function/pom.xml

```diff
@@ -2,18 +2,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<artifactId>image-recognition-function</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
-	<name>image-recognition-function</name>
-	<description>Spring Native Function for Tensorflow Integration</description>
 
 	<parent>
 		<groupId>org.springframework.cloud.fn</groupId>
 		<artifactId>spring-functions-parent</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
+		<version>1.1.0-SNAPSHOT</version>
 		<relativePath>../../spring-functions-parent</relativePath>
 	</parent>
 
+	<artifactId>image-recognition-function</artifactId>
+	<name>image-recognition-function</name>
+	<description>Spring Native Function for Tensorflow Integration</description>
+
 	<properties>
 		<commons-io.version>1.3.2</commons-io.version>
```

Comment:
Not related to your change but this is something wrong. This is too old. 
@tzolov , please, take a look here: https://mvnrepository.com/artifact/commons-io/commons-io